dll/ntdll/def/ntdll.spec

```diff
@@ -1202,7 +1206,7 @@
 @ stdcall RtlTraceDatabaseLock(ptr)
 @ stdcall RtlTraceDatabaseUnlock(ptr)
 @ stdcall RtlTraceDatabaseValidate(ptr)
-@ stub -version=0x600+ RtlTryAcquirePebLock
+@ stdcall -stub -version=0x600+ RtlTryAcquirePebLock()
 @ stdcall RtlTryEnterCriticalSection(ptr)
```

Comment:
```suggestion
@ stdcall -stub -version=0x600+ RtlTryAcquirePebLock()
@ stdcall -version=0x601 RtlTryAcquireSRWLockExclusive(ptr)
@ stdcall -version=0x601 RtlTryAcquireSRWLockShared(ptr)
@ stdcall RtlTryEnterCriticalSection(ptr)
```

Reply:
Shouldn't these `RtlTryAcquireSRWLock*()` be 0x601+?